giza/giza/content/newTables/views.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2015 MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from rstcloth.rstcloth import RstCloth
+
+logger = logging.getLogger('giza.content.newTables.views')
+
+
+def render_table(rows):
+    r = RstCloth()
+
+    r.directive(name="list-table", fields=[("header-rows", "1") ])
+    r.newline()
+
+    firstRow = True
+
+    for row in rows.ordered_content():
+    	firstField = True
+	   	
+    	if firstRow:
+    		columnCount = len(row.fields)
+    	else:
+    		checkColumns = len(row.fields)
+    		if checkColumns != columnCount:
+    			raise SyntaxError('each row in a table must have the same number of columns')
```

Comment:
My preference here is that you check this in the model, rather than when you're in the middle of writing data, as the error is a property of the data, rather than as a property of the rendering process

(also this isn't a python syntax error, which makes this exception a bit odd, but fine.)